src/config/interfaces.ts

```diff
@@ -0,0 +1,73 @@
+import { RepoSyncState } from "../models/subscription";
+
+export interface Subscriptions {
+	id?: number;
+	gitHubInstallationId: number;
+	jiraHost: string;
+	selectedRepositories?: null | number[];
+	repoSyncState?: RepoSyncState | undefined;
+	syncStatus?: string | undefined;
+	syncWarning?: string | undefined;
+	jiraClientKey: string;
+	createdAt: Date;
+	updatedAt: Date;
+}
+
+interface FailedInstallationHeaders {
+	"access-control-allow-origin": string;
+	"access-control-expose-headers": string;
+	connection: string;
+	"content-encoding": string;
+	"content-security-policy": string;
+	"content-type": string;
+	date: string;
+	"referrer-policy": string;
+	server: string;
+	"strict-transport-security": string;
+	"transfer-encoding": string;
+	vary: string;
+	"x-content-type-options": string;
+	"x-frame-options": string;
+	"x-github-media-type": string;
+	"x-github-request-id": string;
+	"x-xss-protection": string;
```

Comment:
They were for me when I looked at the response.

Reply:
are we're using all of them?  Headers are normally just optional.  Heck, looking at Axios type definition, they don't even try to type them because each server can send different headers, even with different capitalizations rules.  So Axios just have headers as `any`.  This might be overdoing it just a tad :)